topic_resources/javascript/digital-clock.md

```diff
@@ -0,0 +1,21 @@
+# Digital Clock
+
+### Create a digital clock in your browser!
+
+### Getting Started
+- Create a function which will do all of the work to calculate and show the time. Start by loading this function using the document ready.
+
+  ```
+  $(document).ready( function() {
```

Comment:
What if we just provide them the static html file and/or talk about loading js at the end of the file? As long as the `<script`> tag is at the end of the file (and there's no dynamic content), then the dom would be ready when the script executed.